test/cluster_SUITE.erl

```diff
@@ -1083,15 +1163,20 @@ start_n_nodes(NamePrefix, Count) ->
 -if(?OTP_RELEASE >= 25).
 start_erlang_node(Name) ->
     Name1 = list_to_atom(Name),
-    {ok, _, Node} = peer:start(#{name => Name1,
+    {ok, Peer, Node} = peer:start(#{name => Name1,
                                  wait_boot => infinity}),
```

Comment:
The indentation of this line needs to be adjusted with the change of the line above.